internal/cmd/setup/frontend.go

```diff
@@ -12,12 +12,12 @@ func yarnCheck(app meta.App) error {
 		return err
 	}
 	if _, err := exec.LookPath("yarnpkg"); err != nil {
-		err := run(exec.Command("npm", "install", "-g", "yarn"))
+		err := run(exec.Command("npm", "install", "-g", "yarn@berry"))
 		if err != nil {
 			return fmt.Errorf("This application require yarn, and we could not find it installed on your system. We tried to install it for you, but ran into the following error:\n%s", err)
 		}
 	}
-	if err := run(exec.Command("yarnpkg", "install", "--no-progress")); err != nil {
+	if err := run(exec.Command("yarnpkg", "install", "--silent")); err != nil {
```

Comment:
`--silent` option suppresses the whole output for classic yarn, and this is not good. With the results of some searching and testing, I think we can just remove these options.